Header header
int16[] trace